packages/web/components/complex/paginated-table.tsx

```diff
@@ -156,31 +159,31 @@ export const PaginatedTable = ({
         {virtualRows.map((virtualRow) => {
           const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
           return (
-            <tr
-              key={row.id}
-              className="transition-colors focus-within:bg-osmoverse-700 focus-within:outline-none hover:cursor-pointer hover:bg-osmoverse-800"
-              onClick={() => {
-                router.push(getPoolLink(row.original.queryPool));
-              }}
-            >
-              {row.getVisibleCells().map((cell) => {
-                return (
-                  <td key={cell.id}>
-                    <Link
-                      href={getPoolLink(row.original.queryPool)}
-                      key={virtualRow.index}
-                      passHref
-                      onClick={(e) => e.stopPropagation()}
-                    >
-                      {flexRender(
-                        cell.column.columnDef.cell,
-                        cell.getContext()
-                      )}
-                    </Link>
-                  </td>
-                );
-              })}
-            </tr>
+<tr
+  key={row.id}
+  className="transition-colors focus-within:bg-osmoverse-700 focus-within:outline-none hover:cursor-pointer hover:bg-osmoverse-800"
+>
+  {row.getVisibleCells().map((cell) => {
```

Comment:
why did you move the onClick event from the `tr` tag to the `td` tag?

Reply:
The onClick event was moved from the tr tag to the td tag to allow for more granular control over the click events. This way, different actions can be performed based on which cell in the row was clicked. If the onClick event was on the tr tag, the same action would be performed no matter where in the row was clicked.